csstools/index.html

```diff
@@ -1,786 +1,389 @@
 }
-</pre>
-
-<p>They can also be specified for a linked stylesheet:</p>
-<pre>
-&lt;link rel="stylesheet" media="&lt;media types&gt;
-   and &lt;media features&gt;" href="alternative.css" /&gt;
-</pre>
-
-<p class="notes">
-https://developer.mozilla.org/en/CSS/@media#Media_groups
-https://developer.mozilla.org/en/CSS/Media_queries
-</p>
-     </section>
-    </div>
-
-
-
-
-
-
-    <div class="slide normal">
-     <header><h1>Print Styles</h1></header>
-     <section class="content">
-     <p>All modern browsers support the "print" media type for specifying print styles.
-</p>
-<pre>
+          </code></pre>
+          <p>They can also be specified for a linked stylesheet:</p>
+          <pre><code>
+&lt;link rel="stylesheet" media="&lt;media types&gt; and &lt;media features&gt;"
+      href="alternative.css" /&gt;
+          </code></pre>
+          <p class="notes">
+            https://developer.mozilla.org/en/CSS/@media#Media_groups
+            https://developer.mozilla.org/en/CSS/Media_queries
+          </p>
+        </section>
+
+        <section>
+          <h2>Print Styles</h2>
+          <p>All modern browsers support the "print" media type for specifying print styles.</p>
+          <pre><code>
 @media print {
   body {
     background: white;
   }
+
   #header, #footer {
     display: none;
   }
+
   .section {
     page-break-inside: avoid;
     page-break-after: always;
   }
 }
-</pre>
-
-<p class="notes">
-http://reference.sitepoint.com/css/pagedmedia
-</p>
-     </section>
-    </div>
-
-
-
-
-
-    <div class="slide normal">
-     <header><h1>Resolution Queries</h1></header>
-     <section class="content">
-     <p>Modern mobile browsers support media queries which depend on the <code>device-width</code> and/or
-<code>orientation</code>:
-</p>
-<p>Targeting iPads:</p>
-<pre>
-@media only screen
-  and (min-device-width : 768px)
-  and (max-device-width : 1024px) {
-    img {
-      width: 100%;
-    }
-}
-</pre>
-<p>Read more: <a href="http://css-tricks.com/snippets/css/media-queries-for-standard-devices/">Media Queries for Standard Devices</a>
-<p class="notes">
-</p>
-     </section>
-    </div>
-
-
-
-
-
-    <div class="slide normal">
-     <header><h1>Resolution Queries</h1></header>
-     <section class="content">
-     <p>A more general approach is just to query on <code>min-width</code> and <code>max-width</code>,
-which are calculated based on window size, not device size.
-</p>
-<pre>
+          </code></pre>
+          <p class="notes">
+            http://reference.sitepoint.com/css/pagedmedia
+          </p>
+        </section>
+
+
+        <section>
+          <h2>Resolution Queries</h2>
+          <p>Modern mobile browsers support media queries which depend on the <code>device-width</code> and/or
+          <code>orientation</code>:</p>
+          <p>Targeting iPads:</p>
+          <pre><code>
+@media only screen and (min-device-width : 768px)
+                   and (max-device-width : 1024px) {
```

Comment:
the lines after the first one should move over a bit to the left here, and the last `}` should be flush with the left